simulators/portal-interop/src/main.rs

```diff
@@ -659,11 +662,6 @@ dyn_async! {
             Err(err) => panic!("{}", &err.to_string()),
         }
 
-        // send a ping so client A sees it as a seen/connected node
-        if let Err(err) = client_a.rpc.ping(target_enr.clone()).await {
```

Comment:
Just breezing by without full context: why is this the only one of the three that doesn't also require an `add_enr`?

Reply:
Because this one already has add_enr it just also had ping

Reply:
Yeah, the usage of add_enr / ping was inconsistent, and this one did both.